sale_quotation_product_sync/tests/test_quotation_synchronizer.py

```diff
@@ -68,9 +61,9 @@ def test_price_propagation(self):
         order_copy.action_confirm()
 
         self.assertEqual(order_line_p3.price_unit, 450.0)
-        self.assertEqual(order.amount_total, 900.0)
+        self.assertEqual(order.amount_total, 24400.0)
```

Comment:
I have a hard time understanding this change, probably because products on the template changed? Then wouldn't it be easier to define a new template and keep the totals in line with the products defined in the setup?